WordCounter/src/main/java/edu/utoledo/nlowe/WordCount/WordCounters/BubbleSelfAdjustingWordCounter.java

```diff
@@ -69,67 +25,63 @@ public void encounter(String word)
         if (head == null)
         {
             // There are no words in the list. Start the list now
-            head = new Node(new Word(word));
+            head = new Node<>(new Word(word));
             referenceChanges++;
         }
-        else if (head.getWord().getValue().equals(word))
+        else if (head.getValue().getValue().equals(word))
         {
+            comparisons++;
```

Comment:
This needs to be incremented regardless of the result of the actual comparison. Will be fixed in the next commit.